src/ai/backend/manager/container_registry/base.py

```diff
@@ -27,6 +28,8 @@
 
 log = BraceStyleAdapter(logging.getLogger(__name__))
 
+manager_arch = platform.machine()
```

Comment:
We need to normalize the architecture names to support both macOS and Linux (or different compilers used to build the CPython).